strawberry_django_plus/integrations/guardian.py

```diff
@@ -13,7 +13,11 @@
 from guardian.utils import get_anonymous_user as _get_anonymous_user
 from guardian.utils import get_group_obj_perms_model, get_user_obj_perms_model
 
-from strawberry_django_plus.utils.typing import UserType
```

Comment:
Maybe change `UserType` to be a union of `AbstractBaseUser` and `AnonymousUser` instead?

Reply:
still important?

Reply:
AbstractBaseUser has is_anonymous, so I think there is no problem